tgui/packages/tgui/interfaces/StripMenu.tsx

```diff
@@ -375,25 +383,38 @@ export const StripMenu = (props) => {
                           {slot.additionalComponent}
                         </Button>
 
-                        {alternateAction !== undefined && (
-                          <Button
-                            onClick={() => {
-                              act('alt', {
-                                key: keyAtSpot,
-                              });
-                            }}
-                            tooltip={alternateAction.text}
-                            style={{
+                        {alternateActions &&
+                          alternateActions.map((alternateKey, idx) => {
+                            const alternateAction =
+                              ALTERNATE_ACTIONS[alternateKey];
+
+                            const alternateActionStyle = {
                               background: 'rgba(0, 0, 0, 0.6)',
                               position: 'absolute',
-                              bottom: '0',
-                              right: '0',
+                              overflow: 'hidden',
+                              margin: '0px',
+                              maxWidth: '22px', // 22px is a good width
```

Comment:
Useless comment

Reply:
it's there because I wanted 20 or 25
but they both look bad

this is a comment letting people know why it's not a clean number